src/Channel.ts

```diff
@@ -174,9 +174,10 @@ export default class Channel {
       tileoffset = tilex * x + tiley * y * atlasrow;
       for (let j = 0; j < y; ++j) {
         tilerowoffset = j * atlasrow;
-        for (let k = 0; k < x; ++k) {
-          this.volumeData[i * (x * y) + j * x + k] = volimgdata[tileoffset + tilerowoffset + k];
-        }
+        this.volumeData.set(
+          volimgdata.subarray(tileoffset + tilerowoffset, tileoffset + tilerowoffset + x),
+          i * (x * y) + j * x
+        );
```

Comment:
this one is for when we load from our old atlased-png format and need to create a 3d volume from it.  This is the less-critical path because we aren't trying to animate playback of these.

Reply:
same question applies here I suppose, though it's just addition happening here